app/containers/Document/Document.jsx

```diff
@@ -43,6 +43,11 @@ class Document extends React.Component {
      */
     onRender: proptypes.func,
 
+    /**
+     * Hashed location key from React Router.
+     */
+    routeKey: proptypes.string,
```

Comment:
Hmm, why does `Document` care about routing? What are the circumstances in which the content needs to be re-fetched and both a) `contentKey` hasn't changed and b) `dataIsDirty` is false?

Reply:
Fixed